index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
```

Comment:
lang 속성은 꼭 ko로 잡아주세요~
```suggestion
<html lang="ko">
```